### DISCUSSION TITLE
Scrobble Songs from unkown users

### DISCUSSION THREAD

**Discussion by groenska** [navidrome/navidrome#3529] [Q&A]
Hi!
I'm using Navidrome to publish a podcast via the sharing function. Now I would like to have some statistics of the number of plays each episode gets and tried to scrobble to a maloja instance.

the scrobbling itself is working fine, but only for real users. is there any possibility to scrobble public plays (which show up in the logs as unknown users)?

**Comment by kgarner7** [comment]
All scrobbles must be associated with some user. Your only options would be to:
1. Just provide a public (non-admin) account
2. Provide your own frontend for shares, where when a public user plays a track, the frontend sends a scrobble (somehow) to Navidrome for a generic user
